pkg/settings/setting.go

```diff
@@ -144,6 +144,10 @@ var (
 	// AuthUserSessionTTLMinutes represents the time to live for tokens used for login sessions in minutes.
 	AuthUserSessionTTLMinutes = NewSetting("auth-user-session-ttl-minutes", "960") // 16 hours
 
+	// ChartDefaultURL represents the default URL for the system charts repo. It should only be set for test or
+	// debug purposes.
+	ChartDefaultURL = NewSetting("chart-default-url", "https://git.rancher.io/")
```

Comment:
Can we make the setting for all three repos? not just `rancher/charts`.

Reply:
We _could_, but do we need that?
* [rancher/partner-charts](https://github.com/rancher/partner-charts) contains, according to the repo's own description, applications from independent software vendors. Do we want to enable those charts to be tested via such a setting (whether by Rancher or partner teams), just as we would for our own charts (Fleet, etc)?
* same question about [rancher/rke2-charts](https://github.com/rancher/rke2-charts), although which charts that repo contains is less clear to me: charts to run against RKE2? RKE2 distribution itself (I'd think of RKE2 as a prerequisite to Rancher rather than of a set of charts to be installed by Rancher, but happy to learn more on that)?

Reply:
@weyfonk All three built-in repositories use git.rancher.io and we have a use case where we want to use github.com for CI for all three repositories.